src/System.Text.Utf8/src/System/Text/Utf8/Utf8String.CodePointEnumerator.cs

```diff
@@ -0,0 +1,164 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
This is the old (unchanged except for the name) Enumerator implementation. Current Enumerator is new.